<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDiffusionWeightedVolumeDisplayWidget</class>
 <widget class="qSlicerWidget" name="qSlicerDiffusionWeightedVolumeDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>59</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DWI Display Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="ScalarDisplayGroupBox">
     <property name="title">
      <string>Scalar Display</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="DWIComponentLabel">
        <property name="text">
         <string>DWI Component:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="qSlicerScalarVolumeDisplayWidget" name="ScalarVolumeDisplayWidget" native="true"/>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="DWIComponentSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="DWIComponentSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerScalarVolumeDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerScalarVolumeDisplayWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>DWIComponentSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DWIComponentSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DWIComponentSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DWIComponentSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerDiffusionWeightedVolumeDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ScalarVolumeDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
